src/mx_bluesky/I24/serial/fixed_target/i24ssx_Chip_Manager_py3v1.py

```diff
@@ -1026,30 +894,31 @@ def fiducial(point):
     print("MTR3\t%1.4f\t%i\t%i\t%1.4f" % (rbv_3, raw_3, mtr3_dir, f_z))
     print("Writing Fiducial File", 20 * ("%s " % point))
 
-    lg.info("%s Writing Fiducial File %sfiducial_%s.txt" % (name, param_path, point))
-    lg.info("%s MTR\tRBV\tRAW\tCorr\tf_value" % (name))
-    lg.info("%s MTR1\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_1, raw_1, mtr1_dir, f_x))
-    lg.info("%s MTR2\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_2, raw_2, mtr2_dir, f_y))
-    lg.info("%s MTR3\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_3, raw_3, mtr3_dir, f_y))
-
-    f = open(param_path + "fiducial_%s.txt" % point, "w")
-    f.write("MTR\tRBV\tRAW\tCorr\tf_value\n")
-    f.write("MTR1\t%1.4f\t%i\t%i\t%1.4f\n" % (rbv_1, raw_1, mtr1_dir, f_x))
-    f.write("MTR2\t%1.4f\t%i\t%i\t%1.4f\n" % (rbv_2, raw_2, mtr2_dir, f_y))
-    f.write("MTR3\t%1.4f\t%i\t%i\t%1.4f" % (rbv_3, raw_3, mtr3_dir, f_z))
-    f.close()
+    logger.info(
+        "%s Writing Fiducial File %sfiducial_%s.txt" % (name, param_path, point)
+    )
+    logger.info("%s MTR\tRBV\tRAW\tCorr\tf_value" % (name))
+    logger.info("%s MTR1\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_1, raw_1, mtr1_dir, f_x))
+    logger.info("%s MTR2\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_2, raw_2, mtr2_dir, f_y))
+    logger.info("%s MTR3\t%1.4f\t%i\t%i\t%1.4f" % (name, rbv_3, raw_3, mtr3_dir, f_y))
+
+    with open(param_path / f"fiducial_{point}txt", "w") as f:
```

Comment:
Shouldn't this be

```suggestion
    with open(param_path / f"fiducial_{point}.txt", "w") as f:
```

?